packages/amplify-category-api/src/graphql-transformer/transform-graphql-schema-v1.ts

```diff
@@ -31,15 +30,17 @@ import {
   buildAPIProject,
   getSanityCheckRules,
 } from 'graphql-transformer-core';
-import { hashDirectory } from '../upload-appsync-files';
 import { exitOnNextTick } from 'amplify-cli-core';
 import { searchablePushChecks } from './api-utils';
+import { getTransformerFactory } from './transformer-factory';
 
 const apiCategory = 'api';
 const parametersFileName = 'parameters.json';
 const schemaFileName = 'schema.graphql';
 const schemaDirName = 'schema';
 const ROOT_APPSYNC_S3_KEY = 'amplify-appsync-files';
+const DESTRUCTIVE_UPDATES_FLAG = 'allow-destructive-graphql-schema-updates';
```

Comment:
Maybe we don't need to deal with this right now but we could probably manage this out of a constants file

Reply:
Yeah, I think we've moved all references out, but I'm just focused on getting this change in for the CLI release so we can migrate the API package in the next release.